baselines/webworker.generated.d.ts

```diff
@@ -578,13 +663,16 @@ declare var Headers: {
     new(init?: HeadersInit): Headers;
 };
 
+interface IDBArrayKey extends Array<number | string | Date | IDBArrayKey> {
+}
+
 interface IDBCursor {
     readonly direction: IDBCursorDirection;
-    key: IDBKeyRange | IDBValidKey;
+    readonly key: IDBKeyRange | number | string | Date | IDBArrayKey;
```

Comment:
Looks like `IDBValidKey` became `number | string | Date | IDBArrayKey` - should is be an alias for it?

Reply:
The alias is still there, i have explicitly added it.. but the references to it have been inlined.